todo_app/app.py

```diff
@@ -3,21 +3,87 @@
 from todo_app.flask_config import Config
 
 import todo_app.data.session_items as session_items
+import requests, os, json
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+class Item:
+    def __init__(self,title,id,status):
+        self.title = title
+        self.status = status
+        self.id = id
+
+    def create_card(title, list_category):
+        list_id = get_list_id(list_category)
+        payload = {'key':os.environ.get('TRELLO_KEY'),'token':os.environ.get('TRELLO_TOKEN'), \
+        'idList':list_id}
+        url = api_prefix + '1/cards'
+        card_title = {"name": title}
+        api_response = requests.post(url, params=payload, data = card_title)
+        api_response_list = json.loads(json.dumps(api_response.json()))
```

Comment:
Here we json.loads a json.dump - that is converting JSON to a string and then back again - 
I.e. this line is the same as cutting both of those calls out: `api_response_list = api_response.json()`

Reply:
Funny - it wasn't working initially so I spent a lot of time googling and fixing this. To my surprise, removing both loads and dump have no effect on app! Fixed